### PR TITLE
Fixed bug in DllImport for Windows Desktop

### PR DESCRIPTION
**Pull request by fmrico**
Hi,

I have found a small bug in the Desktop Amd64.

Hope it helps!!

### CODE REVIEW

File: rcldotnet_common/DllLoadUtils.cs [hunk 1]
```diff
@@ -48,8 +48,8 @@ public class DllLoadUtilsFactory {
       [DllImport ("api-ms-win-core-libraryloader-l1-2-0.dll", EntryPoint = "FreeLibrary", SetLastError = true, ExactSpelling = true)]
       private static extern int FreeLibraryUWP (IntPtr handle);
 
-      [DllImport ("kernel32.dll", EntryPoint = "LoadLibrary", SetLastError = true, ExactSpelling = true)]
-      private static extern IntPtr LoadLibrary (string fileName, int reserved = 0);
+      [DllImport("kernel32", SetLastError = true)]
+      private static extern IntPtr LoadLibrary(string fileName);
```
Comment (esteve) [inline rcldotnet_common/DllLoadUtils.cs]: I'd rather keep it with `ExactSpelling`, could you change it to `LoadLibraryW`?

File: rcldotnet_common/DllLoadUtils.cs [hunk 2]
```diff
@@ -176,8 +176,8 @@ public class DllLoadUtilsUWP : DllLoadUtils {
 
     public class DllLoadUtilsWindowsDesktop : DllLoadUtils {
 
-      [DllImport ("kernel32.dll", SetLastError = true, ExactSpelling = true)]
-      private static extern IntPtr LoadLibrary (string fileName, int reserved = 0);
+      [DllImport("kernel32", SetLastError = true)]
+      private static extern IntPtr LoadLibrary(string fileName);
```
Comment (esteve) [inline rcldotnet_common/DllLoadUtils.cs]: Same here.

File: rcldotnet_common/DllLoadUtils.cs (hunk 1 above)
Comment (fmrico) [reply rcldotnet_common/DllLoadUtils.cs]: `LoadLibraryW` is not working for. I changed to `LoadLibraryA` with `ExactSpelling = true`

File: rcldotnet_common/DllLoadUtils.cs (hunk 2 above)
Comment (fmrico) [reply rcldotnet_common/DllLoadUtils.cs]: Same here too

File: rcldotnet_common/DllLoadUtils.cs (hunk 1 above)
Comment (esteve) [reply rcldotnet_common/DllLoadUtils.cs]: `LoadLibraryA` is fine as well, it's weird that the unicode version is not available in `kernel32.dll`. Thanks!

**Comment by esteve** [comment]
Thanks!

**Pull request merged**